src/libs/Altinn.Authorization.Integration/src/Altinn.Authorization.Integration.Platform/ResourceRegister/ResourceGetEndpoint.cs

```diff
@@ -0,0 +1,249 @@
+using System.Text.Json.Serialization;
+
+namespace Altinn.Authorization.Integration.Platform.ResourceRegister;
+
+/// <summary>
+/// Client for interacting with the Altinn Resource Register API.
+/// Provides methods to retrieve resource information.
+/// </summary>
+public partial class ResourceRegisterClient
+{
+    /// <inheritdoc/>
+    public async Task<PlatformResponse<ResourceModel>> GetResource(string id, CancellationToken cancellationToken = default)
+    {
+        List<Action<HttpRequestMessage>> request = [
+            RequestComposer.WithHttpVerb(HttpMethod.Get),
+                RequestComposer.WithSetUri(ResourceRegisterOptions.Value.Endpoint, "/resourceregistry/api/v1/resource/", id),
```

Comment:
Trodde ikke dette endepunktet hadde støtte for Altinn App ressurser med id-prefiks: app_

Men testet og ser det er laget støtte for de